build/download-clis.sh

```diff
@@ -5,16 +5,17 @@
 set -e
 
 CLI_DESTINATION_DIR="${CLI_DESTINATION_DIR:=/usr/local/bin}"
+CLI_VERSION=4.16
```

Comment:
Do you have a reason 4.16 why not `latest`?

Reply:
Specifying a channel will give us more stability while giving us the benefit of z-stream updates. Setting it to `latest` has the potential to grab a new version that's broken.